docs/Systems/Tailwind/UtilityClassReferences/Borders/border-spacing.stories.tsx

```diff
@@ -37,20 +36,12 @@ export const BorderSpacing: StoryFn<{ isReversed: boolean }> = ({
       <table className={utilityClass}>
         <tbody>
           <tr>
-            <td className={classnames("outline-[black] outline-1 outline")}>
-              Tutant
-            </td>
-            <td className={classnames("outline-[black] outline-1 outline")}>
-              Meenage
-            </td>
+            <td className={"outline-[black] outline-1 outline"}>Tutant</td>
```

Comment:
This doesn't even need curlies

Reply:
https://github.com/cultureamp/kaizen-design-system/pull/3609

BAM!